Tests/LanguageTest.cs

```diff
@@ -0,0 +1,81 @@
+﻿using AdvancedTaskPart1.Pages.ProfileOverview;
+using AdvancedTaskPart1.Pages.SignIn;
+using AdvancedTaskPart1.Steps;
+using AdvancedTaskPart1.Utilits;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AdvancedTaskPart1.Tests
+{
+    [TestFixture]
+    public class LanguageTest : CommonDriver
+    {
+        Login loginObj;
+        ProfileMenuTabComponents profileMenuTabComponents;
+        ProfileLanguageOverviewComponent profileLanguageOverviewComponent;
+        LanguageComponent languageComponent;
+        LanguageSteps languageSteps;
+
+        public LanguageTest()
```

Comment:
so everything looks great, but i want you to think about clean up and state! as mentioned earlier 


example before adding language you need to make sure there a no existing languages as this could result in false positive test pass
you will need to think about this for all the scenarios

the state is say that you have a test that is adding a language english, the second time you run this test you should delete any existing language so that the state is reset, this will guarantee that your test does not produce false negatives and passes always
you can assume that test execution order is not always guaranteed
imagine if you run the tests  in parallel mode the all tests will run at the same time
so you need to also make sure you do not wipe out other tests data
hence why when you remove data the best thing to do is delete only data you added for the test to be correct
e.g test 1 added english and french as language test 2 added hindi then when you reset data you already know what language you are adding for each test
there for on reset of the two test you should delete english and french for test 1 and hindi for test 2

So to sum up you should delete all data if there is any before the start of any test and then after each test you should delete the data that a particular test is adding!
This is ensure that upon staring the test pack you have reset state and upon finishing each test deletes their data that they added!

so think about first clean up before any test runs (you delete all data around this feature e.g  before any language test run you delete all languages). Then after each test delete only data that the test has added! (test1 added english and test2 to added french then test1 should only delete english and test2 should delete french only)